gtsfm/frontend/correspondence_generator/correspondence_generator_base.py

```diff
@@ -2,33 +2,47 @@
 
 Authors: John Lambert
 """
+from abc import abstractmethod
+from typing import Any, Dict, List, Optional, Tuple
 
-import abc
-from typing import Dict, List, Tuple
+from dask.distributed import Client
 
-from dask.delayed import Delayed
+from gtsfm.common.image import Image
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.common.pose_prior import PosePrior
+from gtsfm.common.types import CALIBRATION_TYPE, CAMERA_TYPE
+from gtsfm.two_view_estimator import TWO_VIEW_OUTPUT, TwoViewEstimator
 
 
 class CorrespondenceGeneratorBase:
     """Base class for correspondence generators."""
 
-    @abc.abstractmethod
-    def create_computation_graph(
+    @abstractmethod
+    def apply(
```

Comment:
should we choose a different name - run_correpondence_generator? generate_correpondences? (I prefer the former)

Reply:
`estimate_two_view`?